.github/workflows/docker-build-static.yml

```diff
@@ -481,12 +401,12 @@ jobs:
         id: create_release
         with:
           allowUpdates: true
-          artifacts:  bin/armhf-bin/lldap-armhf,
-                      bin/aarch64-bin/lldap-aarch64,
-                      bin/amd64-bin/lldap-amd64,
-                      bin/armhf-bin/migration-tool-armhf,
-                      bin/aarch64-bin/migration-tool-aarch64,
-                      bin/amd64-bin/migration-tool-amd64,
+          artifacts:  bin/aarch64-lldap,
```

Comment:
Should we maybe wrap them in a tar with the web assets as well?

Anyone who downloads the binary needs the web assets as well, and people who need just the web assets can then replace the binary with their own.